account_receipt_sale/models/account_fiscal_position.py

```diff
@@ -0,0 +1,29 @@
+from odoo import models, fields, api
+
+
+class Position(models.Model):
+    _inherit = "account.fiscal.position"
+    receipts = fields.Boolean(string='Receipts')
+
+    @api.model
+    def get_receipts_fiscal_pos(self, company_id=None):
+        if not company_id:
+            company_id = self.env.user.company_id
+        receipt_fiscal_pos = self.search(
+            [
+                ('company_id', '=', company_id.id),
+                ('receipts', '=', True),
+            ],
+            limit=1
+        )
+        if not receipt_fiscal_pos:
+            # Fall back to fiscal positions without company
+            receipt_fiscal_pos = self.search(
+                [
+                    ('company_id', '=', False),
+                    ('receipts', '=', True),
+                ],
+                limit=1
+            )
+
+        return receipt_fiscal_pos
```

Comment:
Visto quanto indicato [qui](https://github.com/OCA/l10n-italy/issues/2722#issuecomment-1272490383) direi che questa parte non serve.
O quantomeno non dovrebbe stare in questo modulo, se vogliamo che lo stesso corrisponda allo scopo di `l10n_it_corrispettivi_sale` della 12.0.

Reply:
The purpose of `account_receipt_sale` is essentially in its [DESCRIPTION.rst](https://github.com/OCA/account-invoicing/blob/828fa64a7b74baf0da56bb75aede0c28d05afcda/account_receipt_sale/readme/DESCRIPTION.rst)
so it includes parts of `l10n_it_corrispettivi` and `l10n_it_corrispettivi_sale`